src/utils/app_names.py

```diff
@@ -22,7 +22,7 @@ class ScenarioNames(StrEnum):
 class MetricNames(StrEnum):
     REVENUE = "Revenue"
     EBITDA = "Ebitda"
-    COST_OF_GOODS = "Cost of goods"
+    COST_OF_GOODS = "Cost of Goods"
```

Comment:
Change this name, the variable is Cost of goods

Reply:
name changed